focus-android/app/src/main/java/org/mozilla/focus/input/InputToolbarIntegration.kt

```diff
@@ -132,44 +132,46 @@ class InputToolbarIntegration(
     @VisibleForTesting
     internal fun observeStartBrowserCfrVisibility() {
         startBrowsingCfrScope = fragment.components?.appStore?.flowScoped { flow ->
-            flow.mapNotNull { state -> state.showStartBrowsingTabsCfr }.ifChanged().collect { showStartBrowsingCfr ->
-                if (showStartBrowsingCfr) {
-                    CFRPopup(
-                        anchor = toolbar.findViewById<AppCompatEditText>(R.id.mozac_browser_toolbar_background),
-                        properties = CFRPopupProperties(
-                            popupWidth = 256.dp,
-                            popupAlignment = CFRPopup.PopupAlignment.BODY_TO_ANCHOR_START,
-                            popupBodyColors = listOf(
-                                ContextCompat.getColor(
-                                    fragment.requireContext(),
-                                    R.color.cfr_pop_up_shape_end_color,
+            flow.mapNotNull { state -> state.showStartBrowsingTabsCfr }
```

Comment:
Only an Indentation change, couldn't avoid it because of detekt's line length issue, unfortunately